src/angular/controllers/ctrl-question-import.js

```diff
@@ -0,0 +1,123 @@
+const app = angular.module('materia')
```

Comment:
Once the new question importer has been put through its paces, this file can be removed.